src/graphql/resolvers/getOfferResolver.js

```diff
@@ -1,16 +1,16 @@
-const GetOfferResolver = (db, args) => {
+const GetOfferResolver = async (db, args) => {
   const params = {
     TableName: "Offer",
     Key: {
-      id: args.id
+      offer_id: args.id
     }
   };
-  return promisify(callback => {
-    db.get(params, callback);
-  }).then(result => {
-    if (!result.Item) return { id: args.id };
-    return result.Item;
-  });
+  let offer = await db.get(params).promise();
+  if (!offer.Item) {
```

Comment:
This is an  error state right? Should we log that somewhere?

Reply:
Or maybe it should just return an empty object?

Reply:
I think this just means that the database didn't find the value with the given params. If that is the case, then I think it should just return an empty object. For now I will do that, but if anyone disagrees, we can change that later.

Reply:
That sounds good to me. In that case a more "javascripty" way of expressing this might be
```
let offer = await db.get(params).promise()
return offer.Item || {}
```